main.go

```diff
@@ -99,13 +106,13 @@ func main() {
 	ctrl.Log.WithName("Backplane Operator version").Info(fmt.Sprintf("%#v", version.Get()))
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
-		HealthProbeBindAddress: probeAddr,
-		LeaderElection:         enableLeaderElection,
-		LeaderElectionID:       "797f9276.open-cluster-management.io",
-		// LeaderElectionNamespace: "backplane-operator-system", // Ensure this is commented out. Uncomment only for running operator locally.
+		Scheme:                  scheme,
+		MetricsBindAddress:      metricsAddr,
+		Port:                    9443,
+		HealthProbeBindAddress:  probeAddr,
+		LeaderElection:          enableLeaderElection,
+		LeaderElectionID:        "797f9276.open-cluster-management.io",
+		LeaderElectionNamespace: "backplane-operator-system", // Ensure this is commented out. Uncomment only for running operator locally.
```

Comment:
Is this comment no longer true? And if the namespace is hardcoded to that then I think we'll have the same problem we had in https://github.com/open-cluster-management/backlog/issues/15203

Reply:
Yes... Good catch! I uncomment this when running it locally using `make run`, and forgot to comment it back. Will fix.